packages/monaco/src/browser/monaco-loader.ts

```diff
@@ -69,6 +73,8 @@ export function loadMonaco(vsRequire: any): Promise<void> {
                     global.monaco.editorCommonExtensions = editorCommonExtensions;
                     global.monaco.modes = modes;
                     global.monaco.cancellation = cancellation;
+                    global.monaco.suggestModel = suggestModel;
```

Comment:
if we use `editor.getContribution` then it should not be necessary, we will need only interfaces

Reply:
Let's do that; I do not have any objections unless it works.